psydac/api/feec.py

```diff
@@ -328,6 +339,330 @@ def dirichlet_projectors(self, kind='femlinop'):
         elif kind == 'linop':
             return tuple(femlinop.linop for femlinop in self._dirichlet_proj)
 
+    #--------------------------------------------------------------------------
+    def LST_preconditioner(self, M0=None, M1=None, M2=None, M3=None, hom_bc=False):
```

Comment:
I would call this method `LST_preconditioners` (plural). Further, I suggest to make the arguments keyword-only in order to avoid potential errors:

```suggestion
    def LST_preconditioners(self, *, M0=None, M1=None, M2=None, M3=None, hom_bc=False):
```